styles/index.css

```diff
@@ -52,3 +116,83 @@
   font-family: var(--heading-sans);
   font-weight: 600;
 }
+.left-corner p {
+  margin-left: 0;
+}
+.blogs-section-container {
+  padding: 30px 88px;
+  font-family: var(--paragraph-sans);
+}
+.blogs-header-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
```

Comment:
If you're using flex column, `align-items: center` also center the content horizontally to the mid-point

Reply:
Yes, I added the align-items: center but if only this, the paragraph still starts from the left-side. So I still let text-align: center stay there. is it because the text alignment of the paragraph set to left by default, even if container itself is centered?